wp-content/themes/pub/wporg-learn-2020/page-content-calendar.php

```diff
@@ -13,7 +13,7 @@
 
 	<main id="main" class="site-main">
 
-		<div class="row align-middle between section-heading section-heading--with-space">
```

Comment:
These classes aren't required on headings where there is no right element, and cause the title to be centered on small devices, which is not desirable.